src/Surfnet/StepupMiddleware/CommandHandlingBundle/Configuration/Command/RemoveInstitutionConfigurationByUnnormalizedIdCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Copyright 2017 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupMiddleware\CommandHandlingBundle\Configuration\Command;
+
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\AbstractCommand;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\ManagementExecutable;
+
+final class RemoveInstitutionConfigurationByUnnormalizedIdCommand extends AbstractCommand implements
```

Comment:
Nit, but it could be something to consider. When a command says "ByUnnormalizedId", I would personally expect its parameters to match this to make its requirements as explicit as possible. In the most extreme case, that would mean changing `$institution` to `$unnormalizedInstitutionConfigurationId`, but something like `$unnormalizedInstitution` would be fine as well given that we use institutions as a natural identifier for InstitutionConfigurations.